bigtable/google/cloud/bigtable/table.py

```diff
@@ -483,6 +485,26 @@ def drop_by_prefix(self, row_key_prefix, timeout=None):
             table_admin_client.drop_row_range(
                 self.name, row_key_prefix=_to_bytes(row_key_prefix))
 
+    def batcher(self, flush_count=FLUSH_COUNT, max_row_bytes=MAX_ROW_BYTES):
```

Comment:
s/batcher/mutationBatcher/

Reply:
@sduskis Please explain this change

Reply:
Please rename the `batcher` method to `mutationsBatcher`